spec/system/support_interface/staff_spec.rb

```diff
@@ -10,6 +10,7 @@
   end
 
   it "allows inviting a user" do
```

Comment:
Do we need a version of this test when Azure AD is disabled? In which case we would expect the user to need to provide a password so they can log in?